wiki/Tournaments/Official_support/en.md

```diff
@@ -2,168 +2,171 @@
+
 ## Benefits
 
 Officially supported tournaments have access to:
 
-- **Profile badge prizes for first-place winners**
-- Assessment for a main-menu banner during major matches (generally quarterfinals and above), subject to availability and criteria (see the [relevant section](#requesting-main-menu-banner-support) below)
-- A listing in the [automatic tournament registrations system](https://osu.ppy.sh/community/tournaments)
-- A dedicated screening process to prevent players with serious integrity infringements (cheating, multiaccounting, tournament bans) from playing
+- A dedicated [screening process](#tournament-screening) to prevent players with serious integrity infringements (cheating, multi-accounting, tournament bans) from playing.
+- [Profile badge prizes](#profile-badges) for first-place winners.
+- Assessment for a [main-menu banner advertisement](#requesting-main-menu-banner-support) during major matches, subject to availability and criteria.
 
-## Expectations
+## Eligibility
 
-### Tournaments
+Community-run tournaments which abide by the following hard criteria are eligible for support:
 
-Community-run tournaments receiving this support are expected to abide by the following general ideals:
+- The tournament series must run no more than four times per year.
+- The tournament must have at least 64 players or 16 teams qualifying, OR operate on a Round of 16 double-elimination format at a minimum. Larger scales and formats are permitted.
+- The tournament is organised and run by an *experienced team of accomplished and reputable volunteers***¹** , **OR**, the tournament *has been run at least once before without receiving rewards*.
+- Staff members must not participate in the tournament themselves at any point, and registered players are unable to later assist as staff either. For our definition on "staff", please refer to the [FAQ](#what-is-considered-staff).
+- The forum thread for the tournament **MUST** contain a clearly visible link in a normal-sized font to the [tournament reports form](https://pif.ephemeral.ink/tournament-reports) as the very last content of the original post.
+  - `https://pif.ephemeral.ink/tournament-reports`
+  - This report form is overseen by the Tournament Committee. We encourage all users—participants and staff alike—to make use of this form where necessary. 
 
-- Remain free and fair
-- Be as transparent as possible
-- Preserve competitive integrity
-- Encourage good sportsmanship
+**¹**: An "experienced" team member is loosely defined as someone who has contributed significantly to the successful running of at least **three** badge-receiving tournaments beforehand, or has been a part of the osu! World Cup team. This may vary at the discretion of the account support team.
```

Comment:
```suggestion
**¹**: An "experienced" team member is loosely defined as someone who has contributed significantly to the successful running of at least **three** badge-receiving tournaments beforehand, or has been a part of a official World Cup volunteer team for any game mode. This may vary at the discretion of the account support team.
```

Otherwise, this sounds like the criteria is limited strictly to OWC helpers/volunteers, which it isn't.